documentation/plantuml/publish_diagnosis_keys.pu

```diff
@@ -0,0 +1,23 @@
+@startuml
+skinparam defaultTextAlignment center
+skinparam noteTextAlignment left
+start
+:**Filter keys**
+
+Keys older than 14 days and in future will be filtered;
+:**Verify publish token**
+
+Creates call to publish-token api;
+:**Adjust risk buckets**;
+note right
+  - There is predefined array of day limits for every bucket: 14, 10, 8, 6, 4, 2, -3.
+  - Day value which is compared towards the array is days between symptomsOnSet and encounter
+  - Negative value means encounter has been before symptoms
+  - Value for SymptomsOnSet has been get from health care
```

Comment:
Grammatically strange. Maybe "has been set by a health care professional when creating the publish token"